sass/partials/layout/_constrain.scss

```diff
@@ -1,15 +1,25 @@
 // @file
 // Styles for constraining the page width.
 
-.l-constrain {
+@mixin constrain($width: em($max-page-width), $padding: true) {
   margin-left: auto;
   margin-right: auto;
-  max-width: em($max-page-width);
-  padding-left: $gutter-width / 2;
-  padding-right: $gutter-width / 2;
+  max-width: $width;
+
+  @if $padding {
+    padding-left: $gutter-width / 2;
+    padding-right: $gutter-width / 2;
+  }
+}
+
+.l-constrain {
+  @include constrain;
+}
+
+.l-constrain--global {
```

Comment:
I don’t see the benefit of adding this variant into Gesso core. Is there a use-case I’m missing that wouldn’t be project-specific?